Classifiers/classifier-SplunkPy.json

```diff
@@ -28,6 +28,19 @@
 				}
 			},
 			"dontMapEventToLabels": false
+		},
+		"Malware": {
+			"internalMapping": {
```

Comment:
we can add "Vendor Product" -> "Splunk"  here as well

Reply:
the vendor is not splunk.. because the vendor is the system the found the malware (like symantec endpoint) splunk is just the log mgmt